tests/topotests/bgp_multi_vrf_topo2/test_bgp_multi_vrf_topo2.py

```diff
@@ -2011,6 +2033,1783 @@ def test_vrf_route_leaking_next_hop_interface_flapping_p1(request):
+                                        "dest_link": {
+                                            "r1-link3": {
+                                                "keepalivetimer": KEEPALIVETIMER,
+                                                "holddowntimer": HOLDDOWNTIMER,
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                    },
+                },
+                {
+                    "local_as": "100",
+                    "vrf": "BLUE_B",
+                    "address_family": {
+                        "ipv4": {
+                            "unicast": {
+                                "neighbor": {
+                                    "r2": {
+                                        "dest_link": {
+                                            "r1-link4": {
+                                                "keepalivetimer": KEEPALIVETIMER,
+                                                "holddowntimer": HOLDDOWNTIMER,
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                        "ipv6": {
+                            "unicast": {
+                                "neighbor": {
+                                    "r2": {
+                                        "dest_link": {
+                                            "r1-link4": {
+                                                "keepalivetimer": KEEPALIVETIMER,
+                                                "holddowntimer": HOLDDOWNTIMER,
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                    },
+                },
+            ]
+        }
+    }
+
+    result = create_router_bgp(tgen, topo, input_dict_4)
+    assert result is True, "Testcase {} :Failed \n Error: {}".format(tc_name, result)
+
+    for addr_type in ADDR_TYPES:
+        clear_bgp(tgen, addr_type, "r1", vrf=["RED_A", "RED_B", "BLUE_A", "BLUE_B"])
+
+    step("Delete vrfs RED_A and BLUE_A from R1.")
+
+    input_dict = {
+        "r1": {
+            "vrfs": [
+                {"name": "RED_A", "id": "1", "delete": True},
+                {"name": "BLUE_A", "id": "3", "delete": True},
+            ]
+        }
+    }
+
+    result = create_vrf_cfg(tgen, input_dict)
+    assert result is True, "Testcase {} :Failed \n Error {}".format(tc_name, result)
+
+    step(
+        "R2 must not receive the prefixes(in respective vrfs)"
+        "originated from RED_1 and BLUE_1."
+    )
+
+    step("Wait for holddowntimer+1 sec..")
```

Comment:
This info msg would be more useful with real values. I mean HOLDDOWNTIMER+1

Reply:
Done